src/css/LocalSvg.tsx

```diff
@@ -39,7 +40,7 @@ export async function loadAndroidRawResource(uri: string) {
 
 export function loadLocalRawResourceAndroid(source: ImageSourcePropType) {
   const uri = getUriFromSource(source);
-  if (isUriAnAndroidResourceIdentifier(uri)) {
+  if (uri && isUriAnAndroidResourceIdentifier(uri)) {
     return loadAndroidRawResource(uri);
   } else {
     return fetchText(uri);
```

Comment:
If there's no URI, `fetchText` should be executed?

Reply:
In that case, when I updated the getUriFromSource [function](https://github.com/software-mansion/react-native-svg/pull/2382/files#diff-e08238a7634331fb1f865645ff2ff331b68fe550dfeb2ee36971661c92a757a3R8-R13), it is required to add a check on line 43 to ensure that the URI exists for some reason.